.github/workflows/html5sortable.yml

```diff
@@ -0,0 +1,29 @@
+name: Build and test 
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Cache node modules
+      uses: actions/cache@v2
+      env: 
+        cache-name: cache-node-modules
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+        restore-keys: |
+          ${{ runner.os }}-build-${{ env.cache-name }}-
+          ${{ runner.os }}-build-
+          ${{ runner.os }}-
+
+    - name: Build and test
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
```

Comment:
From the last Travis CI log:

```
$ node --version
v14.13.1
```

Reply:
sure, @atodorov. You should be able to see it here: https://github.com/Gontrum/html5sortable/runs/1242446082?check_suite_focus=true

I also updated the job to use node version 14.x. Thank you for your feedback.

Reply:
The failing test are happening since I updated some of the dependencies. Somehow though not locally, any ideas why they only happen on travis / github actions?

Reply:
When I run `npm run standard` locally, the error occurs too. 
Do you have some global dependencies installed, which could overwrite local dependencies?

Reply:
Good idea, but somehow not, I uninstalled all global npm modules and still not. And I just cloned the repo.

Reply:
I could give it a try inside a docker-environment, later this day. Maybe we can narrow down the problem this way :-)

Reply:
I got it running, now. Your dependencies where fine but something went wrong with coveralls and travis. But that doesn't matter now, because a different mechanism is used for github. 
Standard failed because of some globals which were not part of the standard configuration. I also added a comment for disabling eslints `no-use-before-define` inside `store.ts` and hope that's fine with you. 

i also had to adjust the readme, because the travis-badge is of course out of date after the update 🙂

Reply:
@atodorov I just realized, that the action is still not running with the pull request. 
I think this is because the workflow is not yet part of the main-branch. 
Since the master build with travis is currently broken and the [github-build in my fork works](https://github.com/Gontrum/html5sortable/runs/1243550206?check_suite_focus=true): do we want to merge and check if it works here?